packages/block-library/src/html/edit.js

```diff
@@ -59,14 +59,14 @@ export default function HTMLEdit( { attributes, setAttributes, isSelected } ) {
 						isPressed={ ! isPreview }
 						onClick={ switchToHTML }
 					>
-						<span>HTML</span>
+						HTML
```

Comment:
Do you know why the translation was omitted here?

Reply:
The original PR (#1391) doesn't say, but I'm guessing that it's because "HTML" in another language would (presumably) still be "HTML".

Reply:
Makes sense.